client/components/data/query-rewind-state/index.jsx

```diff
@@ -0,0 +1,31 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestRewindState } from 'state/rewind/actions';
+
+export class QueryRewindState extends Component {
+	componentDidMount() {
+		this.request();
+	}
+
+	request() {
+		this.props.request( this.props.siteId );
```

Comment:
Do we want to have something here that checks if the siteId actually is not null. or should that be done somewhere else?

Reply:
oh definitely!